src/libraries/System.Net.Http/src/System/Net/Http/HttpClient.cs

```diff
@@ -510,7 +510,9 @@ private void CheckRequestBeforeSend(HttpRequestMessage request)
                 // Wait for the send request to complete, getting back the response.
                 response = async ?
                     await base.SendAsync(request, cts.Token).ConfigureAwait(false) :
+#pragma warning disable CA1416 // Validate platform compatibility, not supported on browser, safe to suppress
```

Comment:
Why does this need to be suppressed?  Shouldn't the whole call chain be `[UnsupportedOSPlatform("browser")]`?

Reply:
You can `SendAsync` on `browser`. You just can't `Send`. All the places that call this method with `async == false` already have `[UnsupportedOSPlatform("browser")]` on them.

Reply:
Ah, I see.